interface/main/main_info.php

```diff
@@ -49,36 +49,7 @@
 } else {
     $framesrc = "calendar/index.php?module=PostCalendar&func=view&viewtype=" . attr_url($viewtype);
 }
-?>
-<!DOCTYPE html>
-<html>
-<head>
-    <title><?php echo xlt('Calendar'); ?></title>
-    <style>
-        iframe {
-            -webkit-overflow-scrolling:touch
-            display: table;
-            empty-cells: show;
-            border-collapse: collapse;
-            border: 0;
-            width: 1px;
-            min-width: 100%;
-            min-height: 100%;
-            height: 100%;
-        }
-        body {
-            overflow:hidden;
-            -webkit-overflow-scrolling:touch;
-            margin: 0;
-            padding: 0;
-            min-width: 100%;
-            min-height: 100%;
-            width: 100%;
-            height: 100vh;
-        }
-    </style>
-</head>
-<body>
-    <iframe name="Calendar" src="<?php echo $framesrc; ?>" allowfullscreen></iframe>
-</body>
-</html>
+
+// Removed frame as it causes framing issues related to height
+// This functions completely normally without it
+header("Location: " . $framesrc);
```

Comment:
since you are no longer echoing the $framesrc into the html, that means all the attr_url() calls in this script can be changed to urlencode() calls